tri.js

```diff
@@ -2,60 +2,441 @@
 
 // variables
 let gameStarted = false;
-var circleArrayGreen = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-var circleArrayBlack = [];
 var resetButton = document.querySelector("#reset");
-var circleOne = document.querySelector("#one");
+var circleOne = document.querySelector("#\\31 ");
```

Comment:
Typo? Should this be:
```suggestion
var circleOne = document.querySelector("#1 ");
```

Reply:
\\31 is how the querySelector  get #1